update

```diff
@@ -10,9 +10,9 @@ foreach ($manifest as $m) {
     if (!in_array($version, $tags, true)) {
         run("git checkout -b dist");
         run("curl -LO $url");
-        run("mv deployer.phar dep");
-        run("chmod +x dep");
-        run("git add dep");
+        run("chmod +x deployer.phar");
+        run("ln -s deployer.phar dep");
```

Comment:
Let’s just change composer.json bin path.

Reply:
Thanks for the review. I have updated the PR.